README.md

```diff
@@ -66,15 +66,25 @@ We have a [wiki](https://github.com/gfx-rs/wgpu/wiki) that serves as a knowledge
 
 ## Supported Platforms
 
-   API   |    Windows                    |  Linux & Android   |    macOS & iOS     |
-  -----  | ----------------------------- | ------------------ | ------------------ |
-  Vulkan | :white_check_mark:            | :white_check_mark: |                    |
-  Metal  |                               |                    | :white_check_mark: |
-  DX12   | :white_check_mark: (W10 only) |                    |                    |
-  DX11   | :construction:                |                    |                    |
-  GLES3  |                               | :ok:               |                    |
-
-:white_check_mark: = First Class Support — :ok: = Best Effort Support — :construction: = Unsupported, but support in progress
+   API   |    Windows                    |  Linux & Android          |    macOS & iOS     |
+  -----  | ----------------------------- | ------------------------- | ------------------ |
+  Vulkan | :white_check_mark:            | :white_check_mark:        |                    |
+  Metal  |                               |                           | :white_check_mark: |
+  DX12   | :white_check_mark: (W10 only) |                           |                    |
+  DX11   | :hammer_and_wrench:           |                           |                    |
```

Comment:
Should probably be both the hammer and wrench and angle.

Reply:
it's tricky. Currently the table talks about wgpu backends, not the *actual* backends. They used to match, but with Angle they don't

Reply:
I think people end up using this table as "what wgpu can run on", so if we get there via angle or via a native backend it only matters to them in that they need to ship angle or not. So maybe let's have it be just the angle marker for now and once dx11 is back in place we can figure out what to do then.

Reply:
I moved Angle to a separate row